package.json

```diff
@@ -14,6 +14,7 @@
     "release:major": "np major",
     "test": "run-s eslint test:unit",
     "test:unit": "mocha",
+    "report": "node_modules/allure-commandline/bin/allure generate .allure-results --clean && node node_modules/allure-commandline/bin/allure open",
```

Comment:
Why not just `allure generate`?

Reply:
for me is important to see report, so I open it. Of course I could add opening report in new command, but faster is to do it at ones

Reply:
@netmajor I think what @just-boris is trying to say here is that you can leave out `node node_modules/allure-commandline/bin/` and just write `allure`. NPM should look into the right directory to find the executable

Reply:
When I don't specify .allure-result directory, it look into allure-directory which don't exist :/
By default report is not open by allure :/

Reply:
you can keep all the parameters , just remove `node_modules/allure-commandline/bin/`

Reply:
ok, i check this

Reply:
I need to remove current fork, because I FORGOT to rebase from master and have now mess :/ I back with better PR

Reply:
I get :
```
Error: Cannot find module '/Users/Netmajor/Documents/wdio-allure-reporter/allure'
```
So I need to leave this path